### PR TITLE
Update collection of pretrained models for RNNT

### PR DESCRIPTION
**Pull request by titu1994**
# Changelog
- Add pretrained checkpoints of RNNT models
  - ContextNet RNNT trained on ASR SET
  - ContextNet RNNT trained on MLS
  - Conformer RNNT trained on ASR SET
  - Conformer RNNT trained on MLS
- Add pretrained model for ECAPA TDNN (Speaker model)

Signed-off-by: smajumdar <titu1994@gmail.com>

### CODE REVIEW

**Review by nithinraok — APPROVED** [review]

File: nemo/collections/asr/models/rnnt_bpe_models.py [hunk 1]
```diff
@@ -34,15 +34,72 @@ class EncDecRNNTBPEModel(EncDecRNNTModel, ASRBPEMixin):
...
+            pretrained_model_name="stt_en_contextnet_1024_mls",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_en_contextnet_1024_mls",
+            location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_en_contextnet_1024_mls/versions/1.0.0/files/stt_en_contextnet_1024_mls.nemo",
+        )
+        results.append(model)
+
+        model = PretrainedModelInfo(
+            pretrained_model_name="stt_en_conformer_transducer_medium_mls",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_en_conformer_transducer_large_mls",
+            location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_en_conformer_transducer_large_mls/versions/1.4.0/files/stt_en_conformer_transducer_large_mls.nemo",
```
Comment (jbalam-nv) [inline nemo/collections/asr/models/rnnt_bpe_models.py:91]: description and url say large_mls, should be medium_mls?

**Review by jbalam-nv — CHANGES_REQUESTED** [review]

File: nemo/collections/asr/models/rnnt_bpe_models.py (hunk 1 above)
Comment (titu1994) [reply nemo/collections/asr/models/rnnt_bpe_models.py:91]: Good catch

Comment (titu1994) [reply nemo/collections/asr/models/rnnt_bpe_models.py]: Removed it until NGC releases the version. FYI @VahidooX

**Review by jbalam-nv — APPROVED** [review]
LGTM

**Pull request merged**